debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/RowDeserializers.java

```diff
@@ -168,6 +172,12 @@ protected Serializable deserializeTimestampV2(int meta, ByteArrayInputStream inp
         protected Serializable deserializeYear(ByteArrayInputStream inputStream) throws IOException {
             return RowDeserializers.deserializeYear(inputStream);
         }
+
+        @Override
+        protected Serializable deserializeNewDecimal(int meta, ByteArrayInputStream inputStream) throws IOException {
```

Comment:
I'm wondering why this is plugged in here. What's the benefit of using `SpecialValueDecimal` here? Wouldn't it be converted into that within `JdbcValueConverters#convertDecimal()` anyways?

Reply:
Oracle also supports `Infinity` and `NaN` so I thought it will be better to have `SpecialValueDecimal` the default datatype for all JDBC connectors reagrdless of special vlaue support or not.

Reply:
Yes, that makes sense. But it seems it will be converted into `SpecialValueDecimal` already in `convertDecimal()`. So no need to do this in the row deserializer, I think.

Reply:
I see you are right, I did not have it originally in `convertDecimal()`. I introduced it back later during bug fixing. So it is superfluous now.